Pytest fails at `test_abinit_parser`

Currently, pytest fails at `test_parse_files` in `test_abinit_parser` with:
```
=========================== short test summary info ============================
FAILED tests/parsers/test_abinit_parser.py::test_parse_file - pint.errors.DimensionalityError: Cannot convert from 'hartree' ([mass] * [length] ** 2 / [time] ** 2) to '1 / joule' ([time] ** 2 / [mass] / [length] ** 2)
================= 1 failed, 14 passed, 387 warnings in 26.14s ==================
```

--- a/src/nomad_simulation_parsers/schema_packages/abinit.py
+++ b/src/nomad_simulation_parsers/schema_packages/abinit.py
@@ -120,7 +120,7 @@ class TotalForce(outputs.TotalForce):
 class ElectronicDensityOfStates(outputs.ElectronicDensityOfStates):
     outputs.ElectronicDensityOfStates.value.m_annotations.setdefault(
         MAPPING_ANNOTATION_KEY, {}
-    ).update(dict(dos=Mapper(mapper='.value', unit='hartree')))
+    ).update(dict(dos=Mapper(mapper='.value', unit='1 / hartree')))
 
 
 class ElectronicBandStructure(outputs.ElectronicBandStructure):

